### PR TITLE
Fixes #7 - sync DOM with scope.model

### LINKED ISSUE #7: DOM and model array are not synced
When I use your example with ngRepeat:
```js
function ($scope) {
    $scope.many = ['The', 'possibilities', 'are', 'endless!'];
  }
```
And I change order or remove some items..
How can I get updated 'many' array?

### PR DESCRIPTION
**Pull request by luckylooke**
Only one problem I see here... if some use filtered ngRepeat. In such case DOM element index will differ from model index. I will solve this another time.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,21 +220,32 @@ <h3 class='tagline'>Drag and drop so simple it hurts</h3>
...
+        &lt;div ng-repeat='text in many' ng-bind='text'&gt;&lt;/div&gt;
+      &lt;/div&gt;
+      &lt;div class='container' dragula='&quot;another-bag&quot;' dragula-model='many2'&gt;
+        &lt;div ng-repeat='text in many2' ng-bind='text'&gt;&lt;/div&gt;        
+      &lt;/div&gt;
+    &lt;/div&gt;
 
-app.controller('RepeatCtrl', ['$scope',
-  function ($scope) {
+app.controller('RepeatCtrl', ['$scope', 'dragulaService',
```
Comment (bevacqua) [inline index.html]: doesn't seem like it needs `dragulaService`

**Comment by bevacqua** [comment]
Looks great

**Comment by bevacqua** [comment]
I made you a collaborator so you can push to `angular-dragula` directly.

File: index.html (hunk 1 above)
Comment (luckylooke) [inline index.html]: Yes, I forgot to remove it... why jshint didn't tell anything? It doesn't check example.js?

Comment (bevacqua) [inline index.html]: Correct. `.jshintignore` ignores `example` directory

Comment (luckylooke) [inline index.html]: Ok I will remove it soon, I recalled why I left it there.. I wanted to try removeOnSpill, but I forgot on it while writing else.. now when I test it I see that if I use ```dragulaService.options($scope, 'another-bag', {removeOnSpill:true})``` it disconnect models. I have to repair it.

**Pull request merged**